docker-compose.yml

```diff
@@ -17,6 +17,35 @@ services:
             APP_SECRET:
     pattern-library:
         image: ${DOCKER_NAMESPACE}/pattern-library:${REVISION_PATTERN_LIBRARY}
+    scholarly-articles_fpm:
+        image: ${DOCKER_NAMESPACE}/content-store:${REVISION_CONTENT_STORE}
+        volumes:
+            - ./.docker/scholarly-articles/data:/app/data/
+        environment:
+            APP_SECRET:
+            DEFAULT_LOCALE:
+            SERVICE_NAME: scholarly-articles
+    scholarly-articles_web:
+        image: nginx:1.15.2-alpine
+        volumes:
+            - .docker/scholarly-articles/nginx.conf:/etc/nginx/conf.d/default.conf
+        healthcheck:
+            test: ["CMD", "nc", "-z", "localhost", "80"]
+            interval: 5s
+        depends_on:
+            - scholarly-articles_fpm
+    api-gateway:
```

Comment:
Can be merged into `web`?

Reply:
Getting a little confusing that some things (those in `web`) require a `Host` header to access, and this doesn't.

Reply:
I would expect `web` to become only `browser_web`, as the publicly facing side separated from the `api-gateway`. Divided on where the `dummy-api` should be exposed, but the backends and frontends layers can be cleanly separated.

Reply:
`dummy-api` could be removed now as `browser` should read from `api-gateway`? Or kept separate, like `pattern-library`.

Reply:
Yes, as stated in the description `dummy-api` needs to separately be migrated - either as a standalone demo (not used at all) or serving one of the `browser` APIs. The latter is the wrong layer however because it's a stand-in for `api-gateway` rather than for a single `content-store`.

Reply:
On the `Host` header: web exposes a single port to browsers, hence it needs `Host`. `api-gateway` discriminates on path, so doesn't need a special host, though it could if we wanted to. What `api-gateway` can't do (being a separate instance) is being forwarded the same host port 80/443 as `web`.